chore(Changelog): Press Esc to cancel selected text (#841)
cpeditor/cpeditor: CHANGELOG.md

-   Configurable text cursor width. (#781 and #795)
-   Now accepted testcases can be auto-unchecked. (#734 and #797)
-   Update icon for macOS. (#838)

> -   Press Esc key to cance selected text. (#786)

Author: ```suggestion
-   Press Esc key to cancel selected text. (#786)
```

-   Fix that the Find/Replace dialog is not floating in i3-wm. (#767)

> -   Fix text selection color when Dark Fusion is selected. (#788)

Reviewer: ```suggestion
-   Fix text selection color when Dark Fusion is selected. (#788)
-   Now entering/exiting the full-screen mode outside of CP Editor (via the OS/DE/WM) is correctly handled. (#833)
```

## Changed

> -   Esc key now cancels selected text, F11 can be used to enter/exit full screen mode. (#786)

Reviewer: ```suggestion
-   Esc key, which was used to exit full-screen mode, now cancels text selection. Now you can use F11 to enter/exit the full-screen mode. (#786)
```
